charts/prometheus-postgresql-alerts/values.yaml

```diff
@@ -0,0 +1,75 @@
+global:
+  additionalRuleLabels: []
+
+defaultRunbookUrl: https://qonto.github.io/database-monitoring-framework/tree/{{chartVersion}}/runbooks/postgresql/{{alertName}}
+rulesGroupName: postgresql.rules
+
+rules:
+
+  PostgreSQLExporterDown:
+    expr: up{} * on (instance) postgres_exporter_build_info{} < 1
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: "{{ $labels.instance }} exporter is down"
+
+  PostgreSQLExporterScraping:
+    expr: avg_over_time(pg_exporter_last_scrape_duration_seconds{}[10m]) > 30
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.instance }} scraping take long time"
+
+  PostgreSQLMaxConnections:
+    expr: sum(pg_stat_connections_count) by (container) * 100 / on (container) pg_settings_max_connections{} > 80
+    for: 10m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.container }} is close from the maximum database connections"
+      description: "{{ $labels.container }} uses {{ $value }}% of database connections"
+
+  PostgreSQLReplicationSlotStorage:
+    expr: pg_replication_slots_available_storage_percent{} < 20
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.slot_name }} on {{ $labels.dbidentifier }} is close to its storage limit"
+      description: "{{ $labels.slot_name }} is using {{ $value }}% of its storage limit."
+
+  PostgreSQLInactiveLogicalReplicationSlot:
+    expr: max(pg_replication_slots_active{slot_type="logical"} < 1) by (server, slot_name)
+    for: 10m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.slot_name }} on {{ $labels.server | stripDomain | stripPort }} is inactive"
+
+  PostgreSQLInactivePhysicalReplicationSlot:
+    expr: max(pg_replication_slots_active{slot_type="physical"} < 1) by (server, slot_name)
+    for: 10m
+    labels:
+      severity: warning
+    annotations:
+      summary: "{{ $labels.slot_name }} on {{ $labels.server | stripDomain | stripPort }} is inactive"
```

Comment:
```suggestion
      summary: "Physical replication slot {{ $labels.slot_name }} on {{ $labels.server | stripDomain | stripPort }} is inactive"
```

Reply:
We choose to not set it in summary to avoid repeating information that are in alert name